projects/ui-particles-angular/src/lib/gio-confirm-dialog/gio-confirm-dialog.component.ts

```diff
@@ -64,5 +68,6 @@ export class GioConfirmDialogComponent {
 
     this.confirmButton = confirmDialogData?.confirmButton ?? 'Confirm';
     this.cancelButton = confirmDialogData?.cancelButton ?? 'Cancel';
+    this.disableCancel = confirmDialogData?.disableCancel ?? false;
```

Comment:
I would have deal with the absence of `cancelButton`, but I suppose it will be a breaking change

Reply:
If it is not defined it is displayed. no BC otherwise it's a mess 😅